client/src/GlobalStyles.js

```diff
@@ -54,6 +61,12 @@ const GlobalStyles = () => {
           margin: 0;
           padding: 0;
         }
+
+        div {
+          font-family: var(--font-robotolight);
+          size: 0.625px;
+          color: var(--contrast-color-lightgrey);
```

Comment:
Do you want all your divs to be like this by default? What about creating a div component with these properties that you can use and reuse several times?